tls-client/README.md

```diff
@@ -91,4 +91,3 @@ This probably means you need to update the contents of the `SSL_CA_PEM` constant
 Another possible reason for this error is a proxy providing a different certificate. Proxies can be used in some network configurations or for performing man-in-the-middle attacks. If you choose to ignore this error and proceed with the connection anyway, you can change the definition of `UNSAFE` near the top of the file from 0 to 1.
 
 **Warning:** this removes all security against a possible active attacker, so use at your own risk or for debugging only!
-
```

Comment:
Make any whitespace changes all together in a separate commit before any other commits.